### ISSUE TITLE
[Bug] Export to bruno collection does not include the docs element

### ISSUE THREAD

**Issue by SiggiSmara** [usebruno/bruno#1615]
Hello,

As the title says: When exporting a collection to a Bruno json the docs elements are not included in the export.  Might also be something to combine with / related to #605

**Comment by fgalaf** [comment]
This issue was probably already solved. I just exported a collection with docs and it's present in json output under the following key.

```
  "root": {
    "docs": "this is docs section for bruno collection"
  },
```

**Comment by sanjai0py** [comment]
Marking this issue as resolved!

**Comment by fgalaf** [comment]
@Ellesedil you're right, my bad. I misunderstood the title of the story and thought it's only about collection docs missing.

**Comment by helloanoop** [comment]
Merged. Thanks @drwelby 
This will be available in `v1.36.0` scheduled to go out on 4 Dec 2024
